hugegraph-dist/release-docs/LICENSE

```diff
@@ -554,3 +555,11 @@ Third party ISC licenses
 
 The following components are provided under the ISC License.
 	(ISC) * jBCrypt (org.mindrot:jbcrypt:0.4 - https://github.com/djmdjm/jBCrypt)
+
+========================================================================
+Third party GPL licenses
+========================================================================
```

Comment:
We can't introduce GPL license in apache projects, refer to https://www.apache.org/legal/resolved.html#category-x